tests/test_likelihood.py

```diff
@@ -30,8 +30,9 @@
 # Python implementations of the likelihood calculations
 
 
-# TODO give this the same signature as the production version
-def log_arg_likelihood(arg, r, Ne):
+def log_arg_likelihood(arg, recombination_rate, Ne=1):
+    if recombination_rate <= 0:
+        raise ValueError("Recombination rate must be > 0")
```

Comment:
Do we want/need to raise an error for a zero recombination rate? There's no mathematical reason for it: the likelihood is positive with zero recombination rate if there are no breaks in the ARG, and zero if there are breaks.

Reply:
I've no real opinion here, just trying to cover the corner cases. How about you follow up here with a commit which deals with the boundaries at zero?

Reply:
You should be able to run ``git pull`` on your local branch and it'll pull the changes from your ``origin`` remote.

Reply:
Thanks, I'll have a look later on this evening, and delete the mutation comment while I'm at it.